omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb

```diff
@@ -317,6 +317,9 @@
 default['private_chef']['opscode-erchef']['search_queue_mode'] = 'batch'
 default['private_chef']['opscode-erchef']['search_batch_max_size'] = '5000000'
 default['private_chef']['opscode-erchef']['search_batch_max_wait'] = '10'
+default['private_chef']['opscode-erchef']['search_auth_enabled'] = false
```

Comment:
@jashaik @PrajaktaPurohit can we make sure to get a PR to the docs site to add these new config options